.github/workflows/smoke.yml

```diff
@@ -91,16 +109,23 @@ jobs:
         az feature register \
           --namespace=Microsoft.ContainerService \
           --name=GPUDedicatedVHDPreview
-        az group create \
-          --name="tpiSmokeTestCluster$GITHUB_RUN_ID" \
-          --location=eastus
         az aks create \
           --resource-group="tpiSmokeTestCluster$GITHUB_RUN_ID" \
           --name="tpiSmokeTestCluster$GITHUB_RUN_ID" \
           --node-vm-size=Standard_NC6 \
           --node-count=1 \
           --aks-custom-headers=UseGPUDedicatedVHD=true \
           --generate-ssh-keys
+    - id: cluster
+      if: ${{ !contains(env.SMOKE_TEST_TARGET, 'gpu') }}
+      run: |
+        az aks create \
+          --resource-group="tpiSmokeTestCluster$GITHUB_RUN_ID" \
+          --name="tpiSmokeTestCluster$GITHUB_RUN_ID" \
+          --node-vm-size=Standard_NC6 \
```

Comment:
`Standard_NC6` is a GPU-enabled VM size. Did you mean e.g. `Standard_A2_v2` as per https://github.com/iterative/terraform-provider-iterative/pull/673#issuecomment-1249515793?

Reply:
Yes, missed that one - thanks!